modules/thrift/src/thrift/server/TConnectedClient.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2006- Facebook
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef _THRIFT_SERVER_TCONNECTEDCLIENT_H_
+#define _THRIFT_SERVER_TCONNECTEDCLIENT_H_ 1
+
+#include <memory>
+#include <thrift/TProcessor.h>
+#include <thrift/protocol/TProtocol.h>
+#include <thrift/server/TServer.h>
+#include <thrift/transport/TTransport.h>
+
+namespace apache
+{
```

Comment:
Likewise here, this is a C++ file, should have .hpp file extension, and for other files too, especially if the coding standards we have for C and C++ are not the same (as per the compliance failures show in this PR)

Reply:
As mentioned several times in this PR already, there are a few upstream files that are duplicated in the module with slight workarounds for Zephyr's lack of `std::thread` support. These files should be mostly ignored as they will be removed when Zephyr has proper support for `std::thread`.

Specifically, in this case, this header filename needs to match that of the upstream header.